tests/console/osc.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+# Author: Gustavo Figueira <gfigueira@suse.com>
```

Comment:
No need to track the author, we have git for that :) But instead you need a maintainer and summary field

Reply:
I don't have a maintainer for it and what kind of summary?

Reply:
All test modules must have a maintainer and summary field. This is also checked within the CI checks by tools/check_metadata.

You can take a look into other test modules as example. As I assume you will be available in the future just but yourself as maintainer, i.e. just change `# Author:` to `# Maintainer: ` :) And the summary should be a short one-sentence english description of why and what the test does, without needing to duplicate a description of the actual test steps. An optional field is `# Tags: ` which can be used to reference tickets, e.g. boo#1234, if for example the reason why you write this test is a bug that was not found yet by automation